### DISCUSSION TITLE
关于绳网社区维护的建议

### DISCUSSION THREAD

**Discussion by L-walnut** [share121/inter-knot#3272] [其他]
规则：
1.不要带节奏
2.不要发布违规内容，包括造谣、色情等
3.不要引战攻击其他绳网用户
4.没有商业
5.不要刷屏

建议：
1.希望审核真正明白大家希望看什么，不希望看什么，做好社区维护
2.希望作者开放更多功能
3.希望大家开放包容，共同打造一个多元化平台
4.希望大家遵守规则，共同维护社区

**Comment by gtx-d** [comment]
支持<img src="https://img-static.mihoyo.com/communityweb/upload/d053724314d0f532cdb1d89d34e9e02d.png" alt="快跑" width="75" >

**Comment by Zou-pig** [comment]
支持<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >
